pwiz_tools/Shared/Common/SystemUtil/FormUtil.cs

```diff
@@ -60,7 +60,26 @@ public static DialogResult ShowDialog(Control owner, Form dialog)
             }
             return dialogResult;
         }
-        
+
+        /// <summary>
+        /// Moves the control's X and Y coordinates according to the X and Y values.
+        /// </summary>
+        public static void Offset(this Control control, int x = 0, int y = 0)

```

Comment:
Do you mean use the new extension function I added below this one? Because the framework doesn't have a Point function that offsets a point and returns the new point. But yes I agree this function could be briefer by piggybacking off the below one.

Reply:
Oops! You're right. I was the one who was confused.